operating/verify-image-signatures.adoc

```diff
@@ -0,0 +1,41 @@
+[id="verify-image-signatures"]
+= Verifying image signatures
+include::modules/common-attributes.adoc[]
+:context: verify-image-signatures
+
+toc::[]
+
+[role="_abstract"]
+You can use {product-title} ({product-title-short}) to ensure the integrity of the container images in your clusters by verifying image signatures against pre-configured keys.
+
+You can create policies to block unsigned images and images that do not have a verified signature.
+You can also enforce the policy by using {product-title-short} admission controller to stop unauthorized deployment creation.
+
+[NOTE]
+====
+* {product-title-short} 3.70 only supports Cosign signatures and Cosign public key signature verification. For more information about Cosign, see link:https://docs.sigstore.dev/cosign/overview[Cosign overview].
+* You must configure signature integration with at least 1 Cosign public key for signature verification.
+* For all deployed and watched images:
+** {product-title-short} fetches and verifies the signatures every 4 hours.
+** {product-title-short} verifies the signatures whenever you change or update your signature integration public keys.
+====
+
+//Configuring signature integration
+include::modules/configure-signature-integration.adoc[leveloffset=+1]
+
+//Using signature verification
+include::modules/use-signature-verification.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../operating/manage-security-policies.adoc#create-policy-from-system-policies-view_manage-security-policies[Creating a security policy from the system policies view]
+* xref:../operating/manage-security-policies.adoc#policy-criteria_manage-security-policies[Policy criteria]
+
+== Enforcing signature verification
+To prevent the users from using unsigned images, you can enforce signature verification by using the {product-title-short} admission controller. To do this, you must first enable the *Contact Image Scanners* feature in your cluster configuration settings. Then you can use the *Inform and enforce* option while creating a security policy to enforce signature verification.
```

Comment:
Rewrite as: "While creating a security policy to enforce signature verification, you can then use the *Inform and enforce* option.